### DISCUSSION TITLE
Object track list effect on processing load

### DISCUSSION THREAD

**Discussion by cagivacode** [blakeblackshear/frigate#10756] [Ask A Question] [answered]
### What is your question:

a couple of quick questions for @NickM-27 about objects. I am having issues with white tail deer detection.

1). Does listing more objects to track effect either detector or cpu load (or inference speed)?
     e.g. going from person:dog  to person:dog:cat:bird

2). Does adding a max_area filter to an object make it more likely for another object to be detected?
     If I track the "bird" object...almost everything is detected as a bird. If I limit the max_area of the "bird" object does it become 
     more likely to detect the same object as a "dog". Does a ladder of min_area/max_area make any sense. Or, does the model 
     return what it returns and frigate filters it out afterwards.

3). Just an observation because I just had it happen: I removed the "zebra" object from object:track, but forgot that I had also 
    added "zebra" to a zone. In this situation frigate will not run...and there is no error thrown. 
    
   In other words: Frigate will not run with an object assigned to a zone if it is not also being tracked.

regards

**Comment by NickM-27** [comment, marked as answer]
1. The act of adding an object to the list does not affect the usage. However, once one of those objects is detected then it will continue running detection and tracking which naturally will use more cpu usage than if you did not. 

2. No, filters will not cause another labor to take priority. All objects are always returned from the detector. 

3. An error is thrown, there is an error dedicated to this https://github.com/blakeblackshear/frigate/blob/7e5eb82882c3d05a87b501444564a42c6ea53729/frigate/config.py#L1093

That is on purpose

**Comment by cagivacode** [comment]
Thank you...Got it...

So if I filter an object using max_area it will stop tracking it (as that object) allowing it to be tracked as a different object?

or will it just stop tracking that motion/detection all together?

**Reply by NickM-27** [reply]
it will just stop detecting it
